common/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -14,7 +14,7 @@ do_install_bmc_issue () {
     fi
 
     # found out the source dir
-    dir=$(pwd)
+    dir=${COREBASE}
```

Comment:
This logic has been moved to:
https://github.com/facebook/openbmc/blob/helium/classes/openbmc_version.bbclass#L8

If you can send a new pull request based on the latest code, happy to pull in the fix. 

Thx for sending the request.

Reply:
I guess the request can be closed since the updated project code doesn't require these fixes any more.